launch/kinova_vision.launch.py

```diff
@@ -201,6 +201,11 @@ def launch_setup(context, *args, **kwargs):
                     ("depth/camera_info", "depth/camera_info"),
                     ("depth/image_rect", "depth/image_raw"),
                 ],
+                parameters=[
```

Comment:
Might be worth adding a comment that just says exactly what you have in the description...

```suggestion
                # Due to the resolution disparity between the depth images and the rgb images from the embedded DH410 rgbd camera on the gen3 kinova,
                # when the depth images are registered in the color image frame, many sampling holes occur.
                # Turning on the fill_sampling_holes parameter spreads the lower resolution depth data out across the
                # registered image so there are no sampling holes.
                parameters=[
```